src/functions.js

```diff
@@ -0,0 +1,75 @@
+const header = document.getElementById("header");
+const footer = document.getElementById("footer");
+
+const homeContent = document.getElementById("home-content");
+const classesContent = document.getElementById("classes-content");
+const teachersContent = document.getElementById("teachers-content");
+const studentsContent = document.getElementById("students-content");
+
+export function renderNavbar() {
+  return (header.innerHTML = `
+    
+      <nav class="navbar navbar-expand-sm bg-body-tertiary">
```

Comment:
Think about more functions, small functions!

Reply:
Frankly, I thought of finishing the job as a priority. There are actually things (including the structure) that I'm considering changing.